src/General/SettingsGeneral.php

```diff
@@ -143,6 +149,9 @@ static function ($item, $key) {
 
 		$successRedirectVariation = UtilsSettingsHelper::getSettingValue(SettingsGeneral::SETTINGS_GENERAL_SUCCESS_REDIRECT_VARIATION_KEY, $formId);
 
+		$rateLimit = \intval(UtilsSettingsHelper::getSettingValue(Security::RATE_LIMIT_SETTING_NAME, $formId));
+		$rateLimit = ($rateLimit > 0) ? $rateLimit : '';
```

Comment:
```suggestion
		$rateLimit = ($rateLimit > 0) ? $rateLimit : 0;
```

Should this be set to a default integer, if you are casting the setting and checking against integer?

Reply:
I am setting this to an empty value so that the field is empty if a granular form limit isn't set or is invalid